components/delivery-postcode.stories.js

```diff
@@ -9,7 +9,7 @@ export default {
 		country: {
 			control: {
 				type: 'inline-radio',
-				options: ['USA', 'CAN', 'Other'],
+				options: ['USA', 'CAN', 'FRA', 'IND'],
```

Comment:
I guess `Other` was aim to cover uncovered country codes
```suggestion
				options: ['USA', 'CAN', 'FRA', 'IND', 'Other'],
```

Reply:
But I think we should include separate stories for each region, and move USA, CAN option to Default ones with GBR